array-manipulation-1/src/compact.ts

```diff
@@ -0,0 +1,17 @@
+/* exported compact */
+function compact(array: unknown[]): unknown[] {
+  const compactedArr = [];
+  for (const element of array) {
+    if (
+      element !== false &&
+      element !== null &&
+      !Number.isNaN(element) &&
+      element !== 0 &&
+      element !== undefined &&
+      element !== ''
```

Comment:
All of these are considered something called "falsy". Essentially we checking to see if a value evaluates to either true or false. So we can actually just do `if (element)`. That will check to see if `element` gets evaluated to either true or false